server/routes/updateGoal/updateGoalController.ts

```diff
@@ -30,11 +31,25 @@ export default class UpdateGoalController {
       }
 
       updateGoalForm = toUpdateGoalForm(goalToUpdate)
+      updateGoalForm.targetCompletionDate = '2024-02-29'
```

Comment:
This line is a temporary line just to make sure that the goal being updated has a target completion date and therefore the screen controls etc work (otherwise it would cr@p out with a null pointer exception or similar)

Once the API changes mentioned in the description of this PR have been completed then this line will be removed